docs/schema/concepts.rst

```diff
@@ -5,112 +5,97 @@ Key concepts
+As real-world beneficial ownership changes, updated details about people, entities and relationships are submitted to the data management system. The system publishes a new BODS Statement, containing the relevant ``recordId``, for each record whose details are updated or confirmed.
 
-An ``ownershipOrControlStatement`` connects (via ``statementID`` references) the ``subject`` of the statement (an entity described by an ``entityStatement``) and an ``interestedParty``. This interested party may be: 
+People can then use ``recordId`` values to group Statements made over time to see what information was known when. 
 
-* Another entity described by an ``entityStatement``;
-* A person described by a ``personStatement``; or
-* An unspecified party and the reasons that no ownership or control can be specified. 
+.. figure:: ../_assets/key-concepts-img3.svg
+   :alt: A person node is connected via a solid line to an entity node. A relationship statement block sits next to the solid line. A further two relationship statement blocks sit on top of the first. All three relationship statements have recordId 'r-d67fb1a-aa2f3da' and the recordDetails show that these statements are about a shareholding interest. The earliest relationship statement has statementDate '2018-07-12' and recordStatus 'new'. The next has statementDate '2019-11-01' and recordStatus 'updated'. And the final relationship statement has statementDate '2021-02-28' and recordStatus 'closed'.
+   :figwidth: 90%
+   :align: center
 
-The ownership-or-control statement also contains an array of ``interests``, each with a type (selected from the `Interest Type codelist <reference.html#interest-type>`_) and, where relevant, percentages indicating the size of the interest. 
+Published BODS Statements build a write-only ledger, as new Statements are issued to amend or confirm details contained in older Statements.
 
-To explore the structure of the data model in full use the :doc:`Schema browser <schema-browser>`. Or read the :any:`Schema reference <schema-reference>` for detailed definitions of each object and field. 
+See :ref:`guidance-updating-data` for full requirements.
 
+Representing beneficial owners
+------------------------------
 
-Ownership or control through arrangements
-------------------------------------------
+In a BODS dataset, the fact that a natural person is a beneficial owner of an entity is represented by including this information in the Relationship statement linking the two.
 
-In some cases ownership or control is exercised through:
+It is possible to represent in BODS datasets an entity's declaration that it has no beneficial owners (according to a jurisdiction's definition of a beneficial owner).
```

Comment:
> It is possible to represent in BODS datasets an entity's declaration that it has no beneficial owners (according to a jurisdiction's definition of a beneficial owner).

Maybe cut 'in BODS datasets' or I think it should have commas either side as it kind of breaks up the sentence

Reply:
Done